charts/tfy-configs/files/image-mutation-config.yaml

```diff
@@ -0,0 +1,4 @@
+images: []
```

Comment:
let make this confined to atleast repo - not have to change anything if tag is updated